swift/Samples/ModalContainer/Sources/ModalContainerScreen.swift

```diff
@@ -0,0 +1,53 @@
+import WorkflowUI
+
+/// A `ModalContainerScreen` displays a base screen and optionally one or more modals on top of it.
+public struct ModalContainerScreen<BaseScreen: Screen>: Screen
+{
+
+    /// The base screen to show underneath any modally presented screens.
+    public var baseScreen: BaseScreen
+
+    /// Modally presented screens
+    public var modals: [Modal]
+
+    public init(baseScreen: BaseScreen, modals: [Modal]) {
+        self.baseScreen = baseScreen
+        self.modals = modals
+    }
+
+    public func viewControllerDescription(environment: ViewEnvironment) -> ViewControllerDescription {
+        return ModalContainerViewController.description(for: self, environment: environment)
+    }
+}
+
+extension ModalContainerScreen {
+
+    /// Represents a single screen to be displayed modally
+    public struct Modal {
+
+        public enum Style: Hashable {
+            case sheet
```

Comment:
@bencochran  i have changed the names of ModalStyle to a more generic naming. Sheet == fullScreen and popover == card since you mentioned the naming was more specific to square. I have also added animated boolean as a property which is true by default.

Reply:
@bencochran, @asthatrivedi: What do we think the right names for this modal presentation style stuff? See my comment above.

Reply:
Looking at the documentation mentioned in the above comment and this WWDC [video](https://developer.apple.com/videos/play/wwdc2019/224/) explaining different ModalPresentationStyles, i think we can have the following options:

fullScreen: full screen presentation for both iPhone and iPad.
Sheet: since the term sheet now covers formSheet, pageSheet as well as popovers, I think the card style presentation can be named as sheet. 

wdyt?

Reply:
As discussed in a meeting from Friday, the above suggestion has been accepted and i have incorporated the given changes.